pkg/receiver/gitproviderreceiver/internal/scraper/githubscraper/github_scraper.go

```diff
@@ -58,6 +57,23 @@ type githubScraper struct {
 	mb       *metadata.MetricsBuilder
 }
 
+type PullRequestNode struct {
```

Comment:
I like the abstraction of these types outside of the function 👍🏼

Reply:
I should share created with the original author. There were a bunch of these defined outside the function before, I just conformed the offending lines.